src/EndpointsControllers/departmentController.js

```diff
@@ -0,0 +1,94 @@
+import { DefaultController } from './defaultController.js';
+import { handleQuery, recordExist } from '../Helpers/index.js';
+import sql from 'mysql-bricks';
+
+export class DepartmentController extends DefaultController {
+    constructor () {
+        super('department');
+    }
+
+    create () {
+        return async (req, res) => {
+            const managerID = req.body.actualManagerID;
+
+            if (!(await recordExist(managerID, 'department_manager'))) {
+                return res.status(404).send('Department manager not exist');
+            }
+
+            const query = sql
+                .insert(this._tableName, 'name', 'address', 'contact_number', 'actual_manager')
+                .values(req.body.name, req.body.address, req.body.contactNumber, managerID)
+                .toParams({ placeholder: '?' });
+
+            const { values, err } = await handleQuery(query);
+
+            if (err) {
+                console.log(err);
+                return res.status(500).send(err);
+            }
+
+            res.status(200).json(values[0].insertId);
+        };
+    }
+
+    getBooks () {
+        return async (req, res) => {
+            const fromID = req.body.fromID;
+            const amount = req.body.amount * 1;
```

Comment:
If you want to make sure it's a number, you may just add a `+` before the value:

```suggestion
            const amount = +req.body.amount;
```